rebalance.py

```diff
@@ -7,22 +7,24 @@ def rebalance(student_groups, scores, ave):
     """rebalances the groups of students"""
 
     # average score of the groups
-    ave = int(ave / len(scores))
+    ave = int(ave // len(scores))
```

Comment:
This change could make a difference in actual use (using int division instead of floating point). Additionally, if int division is used, casting to int is redundant. Is this well tested?

Reply:
I could not test this since `Gatorgrouper` is not working correctly but I am going to work with @Lancasterwu to resovle this. An alternative to is to revert the changes and import  `__future__statement`

Reply:
Reverted the changes from // to / and disabled the old-division error locally as I found this was not working when included in the `.pylintrc`. Our team will be working with @Lancasterwu to make test cases and code coverage